docker/runtime/ruby-2.4/event-trigger/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM bitnami/ruby:2.4
```

Comment:
is this small ? (nothing to do in this PR, but it would be good to track if this is minimal enough).

Reply:
Not that small at 404mb, the HTTP Ruby image is using it